### ISSUE TITLE
Incorrect property name

### ISSUE THREAD

**Issue by bsatoris** [elastic/enterprise-search-php#35]
The [SearchRequestParams class](https://github.com/elastic/enterprise-search-php/blob/main/src/AppSearch/Schema/SearchRequestParams.php#L26) in AppSearch has a property called `boost` which I think should be called `boosts`. 

If you attempt to build a query using `boost` you will get an invalid key error. The API wants `boosts`.

```
$params->boosts = new Schema\SearchBoosts();
....
//works
```

compare to

```
$params->boost = new Schema\SearchBoosts();
...
//does not work
```

**Comment by ezimuel** [comment]
@bsatoris thanks for reporting this. There was a typo in the specification. I'll fix it soon.

**Comment by ezimuel** [comment]
Fixed in [v8.7.0](https://github.com/elastic/enterprise-search-php/releases/tag/v8.7.0)

**Comment by groeney** [comment]
Hi @ezimuel could you please have someone look at this [fix](https://github.com/elastic/enterprise-search-python/pull/174) for the same bug in the python client? It is a two character change, hopefully it is a fast approval. It is currently blocking our improving of the elasticsearch deployment.
